.webpack/webpack.base.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const webpack = require('webpack');
+const PACKAGE = require('../platform/viewer/package.json');
```

Comment:
Reminder for myself, if we need to pull from here, then we need to make sure we're doing a build/deploy after a successful lerna version for this to be accurate. I may do this in a separate PR